AWS-CodePipeline/sam-monorepo-pipeline-template/{{cookiecutter.outputDir}}/codepipeline.yaml

```diff
@@ -0,0 +1,518 @@
+AWSTemplateFormatVersion : '2010-09-09'
+Description: >
+  This template deploys a CodePipeline with its required resources.
+
+  The following stages are predefined in this template:
+  - Source
+  - UpdatePipeline
+  - UnitTests
+  - BuildAndPackage (MainGitBranch only)
+  - DeployTest (MainGitBranch only)
+  - IntegrationTests
+  - DeployProd (MainGitBranch only)
+
+  **WARNING** You will be billed for the AWS resources used if you create a stack from this template.
+
+# To deploy this template and connect to the main git branch, run this against the leading account:
+# `sam deploy --config-env pipeline`.
+
+Parameters:
+  ProjectSubfolder:
+    Type: String
+    Default: "{{cookiecutter.monorepo_project_subfolder}}"
+  MonorepoRootStack:
+    Type: String
+    Default: "aws-sam-cli-managed-monorepo-codepipeline-root"
+  # Stage 1 Parameters
+  TestConfigEnvName:
```

Comment:
nit: In the question, this is called "configuration name"

> Select an index or enter the stage 1's configuration name (as provided during the bootstrapping)

Reply:
Apologies, but can you expand on this a bit further? I'm not sure what you mean...

Reply:
The term we use to communicate with customer for `TestConfigEnvName` is called "configuration name." The quote is from the interactive question